pyproject.toml

```diff
@@ -85,7 +85,7 @@ pytest = {version = "^6.2", optional = true}
 coverage = {version = "^5.4", optional = true}
 pytest-cov = {version = "^2.11.1", optional = true}
 
-black = {version = "21.9b0", optional = true}
+black = {extras = ["jupyter"], version = "^22.3.0", optional = true}
```

Comment:
We use black for all linters as below. 
I guess we don't need to specify extras.

If it's really needed. We should specify it like others packages in `[tool.poetry.extras]` section of pyproject.toml to be consistent:
```
jupyter = ["jupyter", "nbconvert"]
style = ["black", "isort", "flake8", "pep8-naming", "flake8-docstrings", "mypy", "types-PyYAML", "codespell", "flake8-bugbear", "flake8-comprehensions"]
....
all-dev = [
    "prophet",
    "torch", "pytorch-forecasting",
    "wandb",
    "click", "semver",
    "Sphinx", "numpydoc", "sphinx-rtd-theme", "nbsphinx", "sphinx-mathjax-offline", "myst-parser", "GitPython",
    "pytest-cov", "coverage", "pytest",
    "black", "isort", "flake8", "pep8-naming", "flake8-docstrings", "mypy", "types-PyYAML", "codespell", "flake8-bugbear", "flake8-comprehensions",
    "click", "semver",
    "jupyter", "nbconvert"
]
```